core/src/main/scala/org/apache/spark/sql/sources/jdbcExtensions.scala

```diff
@@ -137,7 +137,7 @@ object JdbcExtendedUtils extends Logging {
     }
     val tableName = if (dotIndex > 0) table.substring(dotIndex + 1) else table
     try {
-      val rs = conn.getMetaData.getTables(null, schemaName, tableName, null)
+      val rs = conn.getMetaData.getTables(null, schemaName.toUpperCase, tableName.toUpperCase, null)
       rs.next()
```

Comment:
This is a side-affect of skipping normalization of names. We should never need to do such code which is also incorrect when "spark.sql.caseSensitive=true" in SQLConf -- please remove.